sgkit/variables.py

```diff
@@ -671,7 +731,7 @@ def _check_field(
 stat_Garud_h1, stat_Garud_h1_spec = SgkitVariables.register_variable(
     ArrayLikeSpec(
         "stat_Garud_h1",
-        ndim={1, 2},
+        dims=("windows", "cohorts"),
```

Comment:
It looks like the Garud stats are always windowed, so we should change their api doc to reflect that.